x-pack/plugins/apm/server/routes/services/get_service_instance_container_metadata.ts

```diff
@@ -29,19 +29,17 @@ type ServiceInstanceContainerMetadataDetails =
   | undefined;
 
 export const getServiceInstanceContainerMetadata = async ({
-  esClient,
-  indexName,
+  infraMetricsClient,
   containerId,
   start,
   end,
 }: {
-  esClient: ElasticsearchClient;
-  indexName?: string;
+  infraMetricsClient: InfraClient;
   containerId: string;
   start: number;
   end: number;
 }): Promise<ServiceInstanceContainerMetadataDetails> => {
-  if (!indexName) {
+  if (!infraMetricsClient) {
     return undefined;
   }
```

Comment:
Is this check necessary? Isn't `infraMetricsClient` required?

Reply:
No, it was checking for the index before, but if the client is created we don't need to check it